README.md

```diff
@@ -73,6 +73,7 @@ npm pack
 cd ../web-components
 npm pack
 ```
+Alternatively, you can use [this build script](./build-tar.sh) to do the same.
```

Comment:
I think it would be good to re-word this to make the link more accessible (e.g. based on our [link guidance](https://design.sis.gov.uk/styles/content-style/links)).

Maybe something like this?
```suggestion
Alternatively, you can use the [`build-tar.sh` script](./build-tar.sh) to do the same.
```